.github/workflows/binary-ready-veracode-sast-policy-scan.yml

```diff
@@ -1,6 +1,6 @@
-name: Binary Ready - Veracode Static Code Analysis
+name: Binary Ready - Veracode Static Code Analysis - Policy Scan
 
-run-name: Binary Ready - Static Code Analysis - ${{ github.event.client_payload.repository.name }}
+run-name: Binary Ready - Static Code Analysis - Policy Scan - ${{ github.event.client_payload.repository.name }}
```

Comment:
I suspect we want to update the name in veracode-code-analysis.yml, though I'm not sure how to differentiate Policy vs Pipeline there... It may be something we need to split up?